### PR TITLE
Fix file contents retrieval

### PR DESCRIPTION
**Pull request by jeromegamez**
Greetings!

First, thank you for this adapter, it saved me a lot of work already. I am using version 2.x of `google/apiclient` and noticed that the file contents are not retrieved correctly (instead of the contents, I got a Response object.

I fixed this in my project by extending the Adapter, but as I just found your 2.x branch, I would like to propose the change I made here as well :).

:octocat:

### CODE REVIEW

**Comment by cedricziel** [comment]
This would be a good moment to remove the `RC` requirement, wouldn't it?

**Comment by jeromegamez** [comment]
Indeed, it would :)

**Comment by matthewgoslett** [comment]
Thanks, taking a look at this now.
It looks like they made a change between RC and V2 which broke this.

File: src/GoogleStorageAdapter.php [hunk 1]
```diff
@@ -262,13 +265,17 @@ public function read($path)
...
         $object = $this->getObject($path);
-        $object = $this->normaliseObject($object);
-        $contents = $this->service->objects->get($this->bucket, $path, ['alt' => 'media']);
-        if ($contents === false) {
-            $contents = null;
+
+        try {
+            $response = $this->service->getClient()->execute(new Request('GET', $object->getMediaLink()));
+        } catch (\Exception $e) {
+            return false;
```
Comment (matthewgoslett) [inline src/GoogleStorageAdapter.php]: Do we want to "mask" the exception here and just return false, or should we just let the exception bubble up?

**Comment by matthewgoslett** [comment]
We've released v3.0.0 which uses the new google/pubsub package, instead of the standard php api client.
This should be resolved with an update.

The instantiation of the adapter has however changed (slightly).

**Pull request closed without merging**